sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -1094,49 +1094,64 @@ func getIPAndPort(address *sensor.NetworkAddress) net.NetworkPeerID {
 	return tuple
 }
 
+func getConnection(conn *sensor.NetworkConnection) *connection {
```

Comment:
I feel we can do better with the function name or the param types here:
It's called `getConnection`, takes a `NetworkConnection` and returns a `connection`.
For someone not intimately familiar with NWG or Collector, I am having trouble instantly grasping what this function does. 
We're determining directionality and extracting some information from a  `NetworkConnection`  (source & destination IP/Port , involved container), but that's something different that getting a connection, no?

Reply:
Yeah, I know, it's terrible. I'll figure out a better name for it.

Reply:
Was torn between `cookConnection` and `processConnection`, went with the boring option for "serious points", which is totally not a thing I just made up.

Reply:
Ahh, that's what being a professional:tm: means, I see :laughing: 
Thanks for improving!